### PR TITLE
feat: add includes and excludes config options

### PR DESCRIPTION
**Pull request by alirezahematidev**
Add two new configuration options, "includes" and "excludes", both of which can be arrays of string regex patterns. These options are used to filter the services based on their service names. This feature is particularly useful when you need to include or exclude specific services from a Swagger endpoint.

for example this swagger config, only generates services that their service name starts with "get":

```
{
  "url": "https://petstore.swagger.io/v2/swagger.json",
  "reactHooks": true,
  "dir": "./src/services",
  "generateEnumAsType": true,
  "includes": ["^get"]
}
```

### CODE REVIEW

File: src/javascript/generator.mts [hunk 1]
```diff
@@ -48,10 +49,41 @@ function generator(
...
+        return matchesInclude && !matchesExclude;
+      },
+    );
+  }
+
   try {
     Object.entries(input.paths).forEach(([endPoint, value]) => {
       const parametersExtended = value.parameters as Parameter[] | undefined;
-      Object.entries(value).forEach(
+      getModifiedPathValue(endPoint, value).forEach(
```
Comment (hosseinmd) [inline src/javascript/generator.mts:86]: `getModifiedPathValue ` doesn't make sense and could be `getFilteredMethods`

File: src/types.mts [hunk 2]
```diff
@@ -263,6 +263,8 @@ export interface FileConfig {
   /** Default is false */
   keepJson?: boolean;
   reactHooks?: boolean;
+  includes?: string[];
+  excludes?: string[];
```
Comment (hosseinmd) [inline src/types.mts:267]: We need to add it in README

**Review by hosseinmd — CHANGES_REQUESTED** [review]
This project now is using by communities, So we can't code without test. Please add a test for your code

File: src/javascript/generator.mts (hunk 1 above)
Comment (alirezahematidev) [reply src/javascript/generator.mts]: getModifiedPathValue changed to getFilteredMethods

**Comment by alirezahematidev** [comment]
also write some tests for includes and excludes new config options

**Comment by hosseinmd** [comment]
swagger-typescript v6.9.0

**Pull request merged**